### DISCUSSION TITLE
Allow custom assertion methods in `php_unit_test_case_static_method_calls`

### DISCUSSION THREAD

**Discussion by jkobus** [PHP-CS-Fixer/PHP-CS-Fixer#8159] [Feature ideas]
## Feature request

It seems that assertion methods provided by symfony, like `assertResponseStatusCodeSame` are ignored.  
It would be handy to be able to define custom assertion methods in the configuration.    

What do you think?

**Comment by keradus** [comment]
converted to issue
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/9190
